funcx_sdk/funcx/sdk/executor.py

```diff
@@ -226,17 +334,97 @@ def _submit_tasks(self, messages):
                 raise
             else:
                 for i, msg in enumerate(messages):
-                    self._function_future_map[batch_tasks[i]] = self._tasks.pop(
+                    task_uuid: str = batch_tasks[i]
+                    self._function_future_map[task_uuid] = self._tasks.pop(
                         msg["task_id"]
                     )
-                    self.poller_thread.atomic_controller.increment()
+                assert self.poller_thread is not None  # Satisfy linter
+                self.poller_thread.atomic_controller.increment(val=len(messages))
+
+    def reload_tasks(self) -> t.Iterable[Future]:
+        """
+        Load the set of tasks associated with this Executor's Task Group (FuncXClient)
+        from the server and return a set of futures, one for each task.  This is
+        nominally intended to "reattach" to a previously initiated session, based on
+        the Task Group ID.  An example use might be:
+
+            import sys
+            import typing as T
+            from funcx import FuncXClient
+            from funcx.sdk.executor import FuncXExecutor
```

Comment:
Nice example, I tried running it so just saw a few adjustments to it

needs `from concurrent.futures import Future` here

Reply:
Ah!  Nice thorough check there.  I'll add that.  Thanks!